### ISSUE TITLE
The umldoclet seems to suppress javadoc errors

### ISSUE THREAD

**Issue by bertbaron** [talsma-ict/umldoclet#13]
When invalid javadoc is encountered by the maven build, the build fails with an error, but the error that caused the issue is not printed when the uml doclet is used. Without the uml doclet a clear error is printed about the invalid javadoc.

Maven version: 3.3.3
JDK version: 1.8.0_74

**Comment by sjoerdtalsma** [comment]
Could you please provide an example of a piece of invalid JavaDoc that triggers this behavior?

**Comment by bertbaron** [comment]
[umldocklet-issue.tar.gz](https://github.com/talsma-ict/umldoclet/files/163097/umldocklet-issue.tar.gz)

With the configuration section of the javadoc plugin commented out, the following error appears in the output of mvn clean package:

```
[ERROR] Exit code: 1 - /home/bert/dev/other/umldocklet-issue/src/main/java/example/JavadocError.java:6: error: invalid use of @return
[ERROR] * @return some field
```
With the umldoclet enabled however this error is gone.

**Comment by sjoerdtalsma** [comment]
The bug that occurred in 0.1.3 has been fixed. That was a NullPointerException that could sometimes happen within the UMLDoclet itself.
So since 0.1.4 the project you uploaded does not seem to throw the meaningless error.
The error without description therefore seems to have been fixed.

However, I find the behavior weird, that the Standard doclet seems to throw the above error, while using the 0.1.5 doclet version accepts your code without any problems.

I think it too should throw the same exception as the Standard doclet throws without the UMLDoclet.
Therefore, I will not close this issue yet, but probably will also not get round to fixing it before the 'First Release' Milestone.

**Comment by sjoerdtalsma** [comment]
I still have not found a solution for this issue.
However, there is a **workaround** by adding option `-umlSkipStandardDoclet true` to the doclet and _afterwards_ performing another (standard) javadoc run.
In maven you can easily configure this as follows:

```xml
<build>
    <plugins>
        <plugin>
            <artifactId>maven-javadoc-plugin</artifactId>
            ...
            <executions>
                <execution>
                    <!-- Call umldoclet to generate UML documentation. -->
                    <id>attach-uml-to-javadocs</id>
                    <goals>
                        <goal>jar</goal>
                    </goals>
                    <configuration>
                        <doclet>nl.talsmasoftware.umldoclet.UMLDoclet</doclet>
                        ...
                        <additionalparam>
                            -umlSkipStandardDoclet true
                            ...
                        </additionalparam>
                    </configuration>
                </execution>
                <execution>
                    <!-- Call standard javadoc target for HTML documentation. -->
                    <id>attach-javadocs</id>
                    <goals>
                        <goal>jar</goal>
                    </goals>
                </execution>
            </executions>
        </plugin>
    </plugins>
</build>
```

**Comment by sjoerdtalsma** [comment]
Needs a re-test to see whether this is still an issue in real projects.

**Comment by sjoerdtalsma** [comment]
This issue seems resolved (at least when testing with jdk 1.8 in our projects) with version `1.0.13`.
Please reopen if the issue turns out to persist.
